test/cql-pytest/test_group_by_limit.py

```diff
@@ -0,0 +1,64 @@
+#
+# Copyright (C) 2022-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+import asyncio
+import time
+import pytest
+import logging
+from cassandra.protocol import InvalidRequest, ReadFailure                         # type: ignore
+from cassandra.query import SimpleStatement                                        # type: ignore
+
+
+logger = logging.getLogger(__name__)
+
+@pytest.mark.asyncio
+async def test_add_index(cql):
```

Comment:
I guess the name was copy-pasted from some other test and needs to be adjusted.

Reply:
Also, please provide an in-source description of what the test does, it's not immediately apparent why choosing some prime numbers etc. Ideally you'd wrap parts of the code into well-named functions/malbdas.